pkgs/games/quakespasm/vulkan.nix

```diff
@@ -34,19 +56,16 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
   '';
 
-  makeFlags = [ "prefix=$(out) bindir=$(out)/bin" ];
-
   env = lib.optionalAttrs stdenv.isDarwin {
     NIX_CFLAGS_COMPILE = "-Wno-error=unused-but-set-variable";
   };
 
   postFixup = ''
+    cp vkquake "$out/bin"
     wrapProgram $out/bin/vkquake \
       --prefix LD_LIBRARY_PATH : ${vulkan-loader}/lib
   '';
 
-  enableParallelBuilding = true;
```

Comment:
Why did you disable this? does it no longer work?

Reply:
`enableParallelBuilding` is set to true by default when using Meson: https://ryantm.github.io/nixpkgs/hooks/meson/